react/features/toolbox/components/native/HangupMenu.tsx

```diff
@@ -29,7 +27,7 @@ import { isInBreakoutRoom } from '../../../breakout-rooms/functions';
  */
 function HangupMenu() {
     const dispatch = useDispatch();
-    const _styles = useSelector(state => ColorSchemeRegistry.get(state, 'Toolbox'));
+    const _styles: any = useSelector(state => ColorSchemeRegistry.get(state, 'Toolbox'));
```

Comment:
Unrelated change.

Reply:
It's related. Because the `ColorSchemeRegistry` import is now fixed TS throws an error saying 'X' does not exists on type StyleType (X being the properties we use from `_styles`)